### ISSUE TITLE
`Deno.inspect(Deno)` throws an exception

### ISSUE THREAD

**Issue by hen-x** [denoland/deno#15704]
Upon running `Deno.inspect(Deno)` in v1.25.0, we see this exception thrown from internal code:

```
error: Uncaught ReferenceError: ArrayPrototypeFilter is not defined
    at Object.ownKeys (deno:ext/node/01_node.js:54:7)
    at keys (<anonymous>)
    at inspectRawObject (deno:ext/console/02_console.js:1100:24)
    at inspectObject (deno:ext/console/02_console.js:1306:30)
    at _inspectValue (deno:ext/console/02_console.js:677:16)
    at inspectValue (deno:ext/console/02_console.js:691:11)
    at inspectValueWithQuotes (deno:ext/console/02_console.js:799:16)
    at inspectRawObject (deno:ext/console/02_console.js:1147:15)
    at inspectObject (deno:ext/console/02_console.js:1306:30)
    at _inspectValue (deno:ext/console/02_console.js:677:16)
```

After debugging, this appears to be a more minimal repro:
```
echo "Reflect.ownKeys(Deno[Deno.internal].node.globalThis)" | deno
Deno 1.25.0
exit using ctrl+d or close()
Uncaught ReferenceError: ArrayPrototypeFilter is not defined
    at Object.ownKeys (deno:ext/node/01_node.js:54:7)
    at Reflect.ownKeys (<anonymous>)
    at <anonymous>:2:9
```

Version info:
```
deno 1.25.0 (release, aarch64-apple-darwin)
v8 10.6.194.5
typescript 4.7.4
```

**Comment by 0f-0b** [comment]
This was fixed in #15595.

**Comment by hen-x** [comment]
Thanks @0f-0b!
